css-view-transitions-1/Overview.bs

```diff
@@ -1589,6 +1588,10 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		* Effects on the element, such as 'opacity' and 'filter' are applied to the capture.
 
+		* Implementations may clip the rendered contents if the [=ink overflow rectangle=] exceeds some [=implementation-defined=] maximum.
+			However, the captured image should include, at the very least, the contents of |element| that intersect with the [=viewport=].
+			Implementations may adjust the rasterization quality to account for elements with a large [=border box=] that are transformed into view.
```

Comment:
Better to say large ink overflow bounds instead of border box. If you have an element with a massive overflow but small box, same policy applies.

Reply:
Done